internal/cli/usercmd/service.go

```diff
@@ -251,3 +253,30 @@ func (c *EpinioClient) ServiceList() error {
 
 	return nil
 }
+
+// ServiceListAll list of all the services instances where the user has permissions
+func (c *EpinioClient) ServiceListAll() error {
+	log := c.Log.WithName("ServiceListAll")
+	log.Info("start")
+	defer log.Info("return")
+
+	c.ui.Note().Msg("Listing all Services...")
+
+	resp, err := c.API.AllServices()
+	if err != nil {
+		return errors.Wrap(err, "service list failed")
+	}
+
+	if len(resp.Services) == 0 {
+		c.ui.Normal().Msg("No services found")
+		return nil
+	}
+
+	msg := c.ui.Success().WithTable("Namespace", "Name", "Catalog Service", "Status")
```

Comment:
As suspected, this is missing the creation timestamps added to the regular list command.